administrator/components/com_content/views/article/tmpl/edit.php

```diff
@@ -82,12 +88,20 @@
 $isModal = $input->get('layout') == 'modal' ? true : false;
 $layout = $isModal ? 'modal' : 'edit';
 $tmpl = $isModal ? '&tmpl=component' : '';
+
+JFactory::getDocument()->addScriptDeclaration('
```

Comment:
You can create the full URL here instead of fragments here and then construct it in the shareDraft() function.